### ISSUE TITLE
ClickHouse - misleading DSN documentation link

### ISSUE THREAD

**Issue by ceckoslab** [benthosdev/benthos#1177]
I was playing with the output for sql_insert with the ClickHouse driver.

From this Benthos page https://www.benthos.dev/docs/components/outputs/sql_insert#drivers I followed an external link that led me to: https://github.com/ClickHouse/clickhouse-go#dsn

The ClickHouse documentation link is about the ClickHouse Go driver V2 where the DSN format has this format:
```
clickhouse://username:password@host1:9000,host2:9000/database?dial_timeout=200ms&max_execution_time=60
```

However, today Benthos uses ClickHouse Go driver V1 where the DSN format has the following format:
```
tcp://host1:9000?username=user&password=qwerty&database=clicks&read_timeout=10&write_timeout=20&alt_hosts=host2:9000,host3:9000
```

I guess as a quick fix we can fix the documentation link and make it pointing to ClickHouse Go driver V1 readme: https://github.com/ClickHouse/clickhouse-go/tree/v1#dsn

**Comment by benhid** [comment]
It appears that Benthos is currently using the `clickhouse-go` driver v2, so I suggest we close this issue to avoid confusion. 👍🏻 @Jeffail

**Comment by mihaitodor** [comment]
Fixed in 0b56956a4 (part of [v4.0.0](https://github.com/benthosdev/benthos/releases/tag/v4.0.0)).
